docs/src/app_dev/app_val_tokenomics/feat_app_act_marker_nomics.rst

```diff
@@ -0,0 +1,83 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _feat_app_act_marker_nomics:
+
+Featured Application Activity Marker
+=====================================
+
+Featured application activity markers
+(`FeaturedAppActivityMarker <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#splice-api-featuredapprightv1>`__)
+are created in a transaction for activity on the network that adds value
+but does not involve a CC Transfer (i.e., transfers a RWA like a USDC).
+A ``FeaturedAppActivityMarker`` is immediately converted into an
+``AppRewardCoupon`` by the automation run by the SuperValidators and the
+``AppRewardCoupon`` is minted just like in a :ref:`cc_xfer_nomics`.
+Please review the application activity marker
+`CIP <https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0047/cip-0047.md>`__
+for details. A summary follows.
+
+There are two prerequisites for an application to create a
+``FeaturedAppActivityMarker``. The first is to become an approved featured
+application which was described in the
+:ref:`app_val_tokenomics`
+section. The second is to update the application code:
+
+   1. Find the fully qualified interface ID for
+   `FeaturedAppRight <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredappright-34177>`__
+   which is
+   ``7804375fe5e4c6d5afe067bd314c42fe0b7d005a1300019c73154dd939da4dda:Splice.Api.FeaturedAppRightV1:FeaturedAppRight``
+   for ``Splice.Api.FeaturedAppRightV1``. The command ``daml damlc
+   inspect-dar`` can be used to find this.
```

Comment:
This is a stable constant. It's probably easiest to just inline this in the follow-up query that you show; and just add a comment that `78043...` is the package-id of the interface definition for the `FeaturedAppRight` interface.

Reply:
Done but didn't inline.